packages/docs/docs/development/css+styleguide.md

```diff
@@ -0,0 +1,61 @@
+# CSS styleguide for Team Arbeidsflate
+
+This minimalistic style guide is designed to ensure consistency and scalability in our frontend projects utilizing CSS Modules. By adhering to these guidelines, we aim to improve maintainability and readability across our codebase.
+It serves to make working with the css easy, and is by no means yet another strict and idiosyncratic book of css full of do's and dont's.
+
+## Naming Conventions
+
+### CSS Classes
+
+- Use **camelCase** for class names to maintain consistency with JavaScript naming conventions.
+
+  ```css
+  /* Example */
+  .profileCard { ... }
+  .userInfo { ... }
+  ```
+
+### CSS Module Files
+Name your CSS module files using the pattern [nameOfComponent].module.css, matching the name of the JSX component they style.
+
+Place the *.module.css file in the same folder as its corresponding JSX component to keep styles closely tied to their components.
+
+A *.module.css file should not be shared among components to ensure styles are encapsulated. The exception is a global module (e.g., global.module.css) for defining global or root variables, which is only indirectly imported.
+
+Encapsulation: Keep styles encapsulated within their respective component module to avoid unintended side effects and maintain component isolation.
+
+Global Styles: Utilize a global CSS module for defining root-level CSS custom properties (variables) and global styles. This approach promotes consistency and reusability across components.
+
+  ```css
+  /components
+    /Button
+        Button.tsx
+        button.module.css
+    /Header
+        Header.tsx
+        header.module.css
+  ```
+### CSS Units
+
+1. For an adaptive design, prefer `rem` over `em` (for simplicity) and `px` (for scalability).
```

Comment:
Could this section be a table instead of a list?

Type|Use|Example|Longer description
-----|----|--------|-------------------
Borders and shadows| Use px | 
Icons and small images | Use either `px` or `%` § 
Zero-sized elements| use `0` instad of `0px`|

Something to make it easy to at a glance see what we want to use, when looking up the docs, without spending to much time re-reading the documentation.

Reply:
Sure. Jeg vil bare ha det så enkelt som mulig slik at en raskt kan bruke 20 sekunder på å slå opp i doc'en. Er enig at vi ikke trenger å liste ut alt, men det må være så enkelt som mulig for øyene å skumme for å finne svaret man er ute etter.